src/composer.json

```diff
@@ -15,7 +15,8 @@
         "symfony/http-foundation": "2.6.*",
         "symfony/dependency-injection": "2.6.*",
         "twig/twig": "1.*",
-        "twig/extensions": "1.*"
+        "twig/extensions": "1.*",
+        "jumbojett/openid-connect-php": "0.1.*"
```

Comment:
this is apparently test files, but building functionality upon Apache-2.0 libs is maybe more difficult. How about asking the author if reliense is possible?